vm/src/vm.rs

```diff
@@ -821,8 +868,37 @@ impl VirtualMachine {
             }
             bytecode::Instruction::CallFunction { count } => {
                 let args: Vec<PyObjectRef> = self.pop_multiple(*count);
-                // TODO: kwargs
-                let args = PyFuncArgs { args: args };
+                let args = PyFuncArgs {
+                    args: args,
+                    kwargs: None,
+                };
+                let func_ref = self.pop_value();
+
+                // Call function:
+                let func_result = self.invoke(func_ref, args);
+
+                match func_result {
+                    Ok(value) => {
+                        self.push_value(value);
+                        None
+                    }
+                    Err(value) => {
+                        // Ripple exception upwards:
+                        Some(Err(value))
+                    }
+                }
+            }
+            bytecode::Instruction::CallFunctionKw { count } => {
+                let kwarg_names = self.pop_value();
+                let args: Vec<PyObjectRef> = self.pop_multiple(*count);
+
+                let kwarg_names = kwarg_names
+                    .to_vec()
+                    .unwrap()
+                    .iter()
+                    .map(|pyobj| pyobj.to_str().unwrap())
```

Comment:
vm.to_str might be the better option here?

Reply:
These are the keyword argument names, which can _only_ be literal strings:

```
$ python -c 'foo("s"+"a"="foo")'
  File "<string>", line 1
SyntaxError: keyword can't be an expression
```

so we shouldn't ever get in to a case where this isn't a string; I've just pushed up a commit to use `objstr::get_value` directly.